### DISCUSSION TITLE
Help With Sk1mmer

### DISCUSSION THREAD

**Discussion by JoeMheh** [3kh0/ext-remover#486] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
I need help with the sk1mmer I was able to get the patched version but don’t know what to do after getting it

**Comment by vernal-ria0** [comment, marked as answer]
how tf you get it it does not even do anything when I click start

**Reply by Aka-but-nice** [reply]
"Skimmer"

**Reply by md4r2** [reply]
@vernal-ria0 It takes a while to start up, but does work. You have to wait for a bit before something comes up.
